### ISSUE TITLE
Service Unavailable

### ISSUE THREAD

**Issue by hellcat121** [Sangwan5688/BlackHole#455]
I am facing "Error Service unavailable" on Spotify charts for the past 5 days now, everything else works flawlessly. First I thought its getting blocked by my ad blocker but I have checked its not blocking it. I don't know if its a bug or your server related issue. I also attaching a screenshot.
I am using Android 12 (oneplus nord)
App version 1.15.3
![Screenshot_2022-11-25-11-21-47-46_a2df05e64545de9e48d383611b669e6e](https://user-images.githubusercontent.com/119164022/204133345-3fc651de-d6d5-4059-82e6-48200a9874ab.jpeg)

**Comment by Sangwan5688** [comment]
duplicate of #460 and fixed
